cmd/gbounty/bootstrap/bootstrap.go

```diff
@@ -670,12 +688,18 @@ func writeScanFromFs(ctx context.Context, w scan.Writer, cfg scan.Config, fs sca
 		}
 	}
 
-	err := w.WriteStats(ctx, fs)
+	var err error
+
+	if !PocEnabled {
+		err = w.WriteStats(ctx, fs)
+	}
 	if err != nil {
 		return err
 	}
 
-	err = w.WriteMatchesSummary(ctx, fs)
+	if !PocEnabled {
```

Comment:
This confuses me, why do we need to pass `PocEnabled` as an argument of this function, if it's only called under one condition?

Reply:
removed it and used the PocEnabled from the NewConsole struct